tests/multihost/container-network/main.fmf

```diff
@@ -0,0 +1,13 @@
+summary: Basic multihost test container networking
+description:
+    Demonstarate and verify that container networking features.
+    This runs a simple webserver from one container and uses curl
+    to read that served content from another container.
+framework: shell
+test: cd data && tmt run -vvv
+
+enabled: false
+adjust:
+  - enabled: true
+    when: how == full
```

Comment:
How about running this each time? With 'container' I don't see the reason why to move it to the 'full' suite only. It shouldn't delay the results too much.

In such case: remove 'enabled' and 'adjust' keys

Reply:
done.